src/vs/base/browser/ui/menu/menu.ts

```diff
@@ -1251,7 +1253,7 @@ ${formatRule(Codicon.menuSubmenu)}
 /* Vertical Action Bar Styles */
 
 .monaco-menu .monaco-action-bar.vertical {
-	padding: .6em 0;
```

Comment:
@sbatten does this affect more than just the menus? The classname is suspicious.

Reply:
the only action bars that are vertical are menus and the activity bar icons I'm pretty sure. but I'm guessing this is overridden by activity bar styles.

Reply:
oh also, since its under .monaco-menu, only menu widgets are affected

Reply:
Ok cool, I'll verify that the change doesn't affect anything else. Let me know if there are any other concerns 👍